### DISCUSSION TITLE
遠距剋星-打瞌睡AI辨識器

### DISCUSSION THREAD

**Discussion by charlottechuang** [1101-GS4719-C/Python-Community-GS4719-C#274] [科技新知]
如題，算是機器學習的應用之一，但也太狠了.....
https://tw.news.yahoo.com/%E5%AD%B8%E7%94%9F%E5%B0%8F%E5%BF%83%E4%BA%86-%E5%8D%97%E9%9F%93%E9%96%8B%E7%99%BC-%E6%89%93%E7%9E%8C%E7%9D%A1ai%E8%BE%A8%E8%AD%98%E7%B3%BB%E7%B5%B1-%E6%88%90%E6%95%99%E5%B8%AB%E8%A9%95%E5%88%86%E6%96%B0%E5%88%A9%E5%99%A8-101425574.html

**Comment by MAKERRRofKEVIN** [comment]
很實際且實用的概念，若能夠維持夠高的精確度便能減少老師課堂評量的難度並提高公平性

**Reply by chia-kai-chang** [reply]
對於課堂評分又可以增加一項量化參考依據，教師可以選擇要不要使用這項數據來決定期末成績。

**Comment by 20010906** [comment]
感覺好恐怖...雖然好處是可成為較公平的評分標準，但都到大學了，應該自我要求，而不是還要依靠AI監督吧，南韓的大學生被如此監督，會不會感到不適？⊙︿⊙

**Reply by chia-kai-chang** [reply]
是啊，有種不舒服的感覺。

**Comment by enoch-lightning** [comment]
我們也要加油，我們的資訊能力絕對不行輸給他們才是。
🐎

**Comment by enoch-lightning** [comment]
我也覺得這樣監督學生的一舉一動不太適合，我不小心一翹個腳就被扣分，我脖子痠，用手枕著也要扣分。

**Comment by zixunn** [comment]
感覺如果學校真的用這種方式監督學生學習的話，立意雖然是為了讓學生更專注於課堂，卻反而會造成反效果，使學生無法在自在放鬆的吸收新知，而是時刻要分心注意自己是否姿儀有違規，並且更加排斥上學。

**Comment by 107602505** [comment]
我覺得這個打瞌睡AI辨識系統可以將其改良成作弊的辨識系統，可以防止學生作弊，對遠距考試的公平性應該會有很大的幫助。

**Reply by Jie2222** [reply]
在疫情期間的確也發展了監考AI，但其實背後也隱藏了很多問題。
[AI監考系統 VS. 線上作弊，誰會贏？](https://www.youtube.com/watch?v=LnasQNcV19A)
[遠距監考的代價：隱私、數位機會均等、電腦自主權](https://www.scimonth.com.tw/archives/5240)

**Reply by 107602505** [reply]
沒想到居然真的有監考AI，感覺未來要有些規範的設立來保障使用者的隱私權

**Comment by JAIMEHUNG** [comment]
我也覺得會影響到那些認真讀書的同學，還要分心去注意自己的坐姿。

**Comment by ghost** [comment]
很實際的應用...大班上課時的確有幫助，不過在台灣的大學其實上課在幹嘛會館的老師好像有不是很多，都大學了應該為自己負責

**Comment by chun1203** [comment]
感覺會讓人有種緊迫逼人的感覺….

**Comment by 109802503gloria** [comment]
感覺這個辨識器也許能用在矯正預防脊椎側彎的不良坐姿上XD

**Comment by mesakiyuta** [comment]
這樣遠距上課壓力會好大，要一直專心上課，除了不能低頭還要時不時動一下，才能符合他的標準，感覺心會好累

**Comment by cody910120** [comment]
出發點是好的，但真的有這必要嗎QQ會偷的還是繼續偷

**Comment by YuHengChen54** [comment]
雖然對老師蠻方便的，但還是有種被監視的感覺不太好

**Comment by Joan0518** [comment]
感覺會造成反效果，學生應該會很反彈吧，真的太狠了...

**Comment by dabbie109303501** [comment]
雖然是滿新穎的發明，但不能只依照頭的傾斜和維持時間就判定有沒有打瞌睡吧？

**Comment by Zoe06060606** [comment]
這太有壓迫感了吧
有種過度應用的感覺

**Comment by ching020721** [comment]
這種AI程式可以作為一種評分方式，但對學生來說可能會因為監視壓迫反而影響學習效果，這種發明是很不錯的想法，但若能增加一些人性與綜合考量供教師參考，就能在科技與人性間獲取平衡。

**Comment by HuangHsinTzu** [comment]
這項技術很適合小學生
小孩的自制力較低，的確需要人監督
可以解決遠距時小孩分心的問題，非常實用!

**Comment by evanchou1204** [comment]
我記得之前有一個產品是開車的時候戴在頭上，如果疲勞駕駛打瞌睡AI會偵測到並且把你叫醒。

**Comment by bonny123865** [comment]
我覺得這個發明可以先不要進來啦!這樣對學生太不友善了吧!
要一直注意自己的姿勢，都不能放鬆一點聽課

**Comment by Doreen331** [comment]
感覺會產生很大的爭議，在偵測的過程不是百分之百準確，不僅對於學生會產生心理壓力，也需要一直分心去在意自己的坐姿，考試已經很累了還要多增加一項需要擔心的事🥲

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
太狠了 雖然可以有效的堅督學生學習 但也有點太不人性化了

**Comment by BlahFish1006** [comment]
個人覺得或許將這個技術應用在矯正姿勢不良會更有意義......
比如加上一個在頭部的小型感測器，輔助判斷姿勢狀態，
提醒看護協助已長時間維持同一姿勢的老人或患者翻身、喚醒打瞌睡的疲勞駕駛、
督促久坐於電腦前的上班族、或長時間窩在電視前面的電視兒童要記得換姿勢動一動，以避免日後出現肩頸痠痛或脊椎側彎問題等等。

**Comment by ghost** [comment]
希望打瞌睡AI不要有實裝的一天，感覺這種監視很像某國的天眼系統，感覺很不舒服。

**Comment by Chiway1229** [comment]
這項技術雖然能讓課堂秩序得以維持，不過可能會對學生心中產生更大的壓力
是有點不合乎人性。

**Comment by evanchou1204** [comment]
我記得之前還看過一種偵測疲勞駕駛的裝置，一旦駕駛睡著AI就會響鈴或震動把駕駛叫醒，這種裝置應用在遠距教學上是真的太狠了.....

**Comment by Evan0979** [comment]
怎麼這麼逆風,我逗覺得這是一個滿有潛力的技術,不必把它局限於辨識打瞌睡!

**Comment by CPCpyt** [comment]
我覺得還滿可怕的，我寧可實體上課坐姿東倒西歪，
另外被監控的感覺對真的身心靈都不好。

**Comment by zephyrxin01** [comment]
我覺得如果作為學生或員工被監控的感覺很不好，但如果應用在車輛駕駛上避免疲勞駕駛可能會有不錯的效果，畢竟這對於行車安全來說很重要！

**Comment by pudding0803** [comment]
但我認為這滿不好的，或許突然發生什麼事情，或是有其他判斷錯誤的可能，都會導致事情變得麻煩，而且監控本身我覺得是不太好的。其實我個人滿容易在沒有意識的情況下不小心睡著的🥲，因此拿打瞌睡來當成認真與否的依據似乎不是太好的做法（還是其實只有我這樣😢？）

**Comment by zhiron544** [comment]
雖然我知道這個的用意是好的，但總覺得這項技術到最後會被拿去用於犯罪。
